internal/containerinsightscommon/nodeCapacity.go

```diff
@@ -18,7 +18,7 @@ type NodeCapacity struct {
 
 func NewNodeCapacity() *NodeCapacity {
 	if _, err := os.Lstat("/rootfs/proc"); os.IsNotExist(err) {
-		panic("/rootfs/proc doesn't exists")
+		panic("E! /rootfs/proc doesn't exists")
```

Comment:
How is this error getting logged? This just adds the telegraf log error prefix but I don't think that this alone writes to a log about the error.

Reply:
That is one of the criteria I mentioned: `most encountered errors has been at surface levels instead of internal package and avoid having unnecessary logs.`. However, if it is needed, I would add logs for this as well.